x/mint/keeper/hooks_test.go

```diff
@@ -395,7 +395,10 @@ func (suite *KeeperTestSuite) TestAfterEpochEnd() {
 
 			osmoassert.ConditionalPanic(suite.T(), tc.expectedPanic, func() {
```

Comment:
We should change the variable here to `expectedError`, and then just do an if statement for checking if error vs checking no error

Reply:
done!